environment.yml

```diff
@@ -0,0 +1,18 @@
+name: numbadev
```

Comment:
I think you committed files that aren't necessary for this PR. Like this file and the `conda-lock.yml`.

Reply:
Oops! Removed